pygments/lexers/qlik.py

```diff
@@ -0,0 +1,139 @@
+"""
+    pygments.lexers.qlik
+    ~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the qlik scripting language
+
+    :copyright: Copyright 2006-2021 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import RegexLexer, include, bygroups, words
+from pygments.token import *
+from pygments.lexers._qlik_builtins import (
+    OPERATORS_LIST,
+    STATEMENT_LIST,
+    SCRIPT_FUNCTIONS,
+    CONSTANT_LIST,
+)
+
+__all__ = ["QlikLexer"]
+
+field_name_options = r"(\w|\[[\w\s]\]|\"[\w\s]\")"
+
+
+class QlikLexer(RegexLexer):
+    """
+    Lexer for qlik code, including .qvs files
+    """
+
+    name = "Qlik"
+    aliases = ["qlik", "qlikview", "qliksense", "qlikscript"]
+    filenames = ["*.qvs", "*.qvw"]
+
+    flags = re.IGNORECASE
+
+    tokens = {
+        # Handle multi-line comments
+        "comment": [
+            (r"\*/", Comment.Multiline, "#pop"),
+            (r"[^\*/]+", Comment.Multiline),
```

Comment:
No need for the slash in the character class here. (It is needed in the doc example because of the support for nested multiline comments.)